docs/airgap-install.md

```diff
@@ -23,7 +23,7 @@ k0s/containerd uses OCI (Open Container Initiative) bundles for airgap installat
 
 k0s offers two methods for creating OCI bundles, one using Docker and the other using a previously set up k0s worker.
 
-**Note:** k0s strictly matches image architecture, e.g. arm/v7 images won't work for arm64.
+**Note:** When importing the image bundle k0s uses containerd "loose" [platform matching](https://pkg.go.dev/github.com/containerd/containerd/platforms#Only). For arm/v8, it will also match arm/v7, arm/v6 and arm/v5. This means that your bundle can contain multi arch images and the import will be done using platfomr compatibility.
```

Comment:
```suggestion
**Note:** When importing the image bundle k0s uses containerd "loose" [platform matching](https://pkg.go.dev/github.com/containerd/containerd/platforms#Only). For arm/v8, it will also match arm/v7, arm/v6 and arm/v5. This means that your bundle can contain multi arch images and the import will be done using platform compatibility.
```